src/helpers/fileNameParser.js

```diff
@@ -0,0 +1,7 @@
+const fileNameRegex = /([0-9]+)\.epub\.((?:no)?images)$/
```

Comment:
The first group could just be anything, `(^.+)`. I would at least add the `^` because if any Gutenberg file ever had something _before_ the numbers this regex would cut it off.